lib/node_modules/@stdlib/blas/base/drot/test/test.drot.js

```diff
@@ -0,0 +1,325 @@
+	t.end();
+});
+
+tape( 'the function applies a plane rotation', function test( t ) {
+	var xexpected;
+	var yexpected;
+	var ixvalues;
+	var iyvalues;
+	var nvalues;
+	var xvalues;
+	var yvalues;
+	var delta;
+	var tol;
+	var out;
+	var ex;
+	var ey;
+	var ix;
+	var iy;
+	var c;
+	var i;
+	var j;
+	var k;
+	var x;
+	var y;
+
+	xexpected = [
+		[ 0.6, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, -0.46, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, -0.46, -0.22, 1.06, 0.9, -0.3, -0.4 ],
+		[ 0.6, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.66, 0.1, -0.1, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.96, 0.1, -0.76, 0.8, 0.9, -0.3, -0.020000000000000073],
+		[ 0.6, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ -0.06000000000000005, 0.1, -0.1, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.9, 0.1, -0.22, 0.8, 0.18, -0.3, -0.020000000000000073],
+		[ 0.6, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, 0.26, -0.5, 0.8, 0.9, -0.3, -0.4 ],
+		[ 0.78, 0.26, -0.76, 1.12, 0.9, -0.3, -0.4 ]
+	];
+	yexpected = [
+		[ 0.5, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.040000000000000036, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.040000000000000036, -0.78, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.040000000000000036, -0.78, 0.54, 0.07999999999999996, -0.6, 0.2, 0.8 ], // eslint-disable-line max-len
+		[ 0.5, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.040000000000000036, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.7, -0.9, -0.12, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.64, -0.9, -0.3, 0.7, -0.18, 0.2, 0.28000000000000014 ],
+		[ 0.5, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.040000000000000036, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.7, -1.08, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.64, -1.26, 0.54, 0.2, -0.6, 0.2, 0.8 ],
+		[ 0.5, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.040000000000000036, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.040000000000000036, -0.9, 0.18, 0.7, -0.6, 0.2, 0.8 ],
+		[ 0.040000000000000036, -0.9, 0.18, 0.7, -0.18, 0.2, 0.16000000000000014 ] // eslint-disable-line max-len
+
+	];
+	nvalues = [ 0, 1, 2, 4 ];
+	ixvalues = [ 1, 2, -2, -1 ];
+	iyvalues = [ 1, -2, 1, -2 ];
+
+	xvalues = [
+		[ 0.6, 0.10, -0.5, 0.8, 0.9, -0.3, -0.4 ]
+	];
+	yvalues = [
+		[ 0.5, -0.9, 0.3, 0.7, -0.6, 0.2, 0.8 ]
+	];
+
+	c = 0;
+	for ( i = 0; i < ixvalues.length; i++ ) {
+		for ( j = 0; j < nvalues.length; j++ ) {
+			x = new Float64Array( xvalues[ 0 ] );
+			y = new Float64Array( yvalues[ 0 ] );
+			ix = ixvalues[ i ];
+			iy = iyvalues[ i ];
+			out = drot( nvalues[ j ], x, ix, y, iy, 0.8, 0.6 );
+			ex = new Float64Array( xexpected[ c ] );
+			ey = new Float64Array( yexpected[ c ] );
+			c += 1;
+			for ( k = 0; k < ex.length; k++ ) {
+				if ( x[ k ] === ex[ k ] ) {
+					t.strictEqual( x[ k ], ex[ k ], 'returns expected value' );
+				} else {
+					delta = abs( x[ k ] - ex[ k ] );
+					tol = 1.5 * EPS * abs( ex[ k ] );
```

Comment:
Not clear to me where the imprecision is coming from given that the implementation is doing simple addition and multiplication.

Reply:
The same loss in precision is observed in fortran output as well when compiled using gfortran

Reply:
Okay. I see they are performing an approximate equality test. It is fine if we do so, as well. And yes, we can just tweak `tol` (e.g., `1.5 * EPS` or `2.0 * EPS`, etc).

Reply:
Better that way than fudging the expected values.

Reply:
Yes, perfect.